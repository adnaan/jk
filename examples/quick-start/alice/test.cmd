jk generate --stdout alice.js
