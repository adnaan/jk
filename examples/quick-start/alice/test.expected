beverage: Club-Mate
languages:
- python
- haskell
- c++
- 68k assembly
monitors: 2
name: Alice
